server/re/src/reDataObjOpr.cpp

```diff
@@ -1540,11 +1542,11 @@ msiDataObjGet( msParam_t *inpParam1, msParam_t *msKeyValStr,
  *            This keyword does not need a value.
  *        \li "ChksumAll" - checksum all replicas. This keyWd has no value.
  *            But the '=' character is still needed.
- *        \li "verifyChksum" - verify the chksum value.
- *        \li "forceChksum" - checksum data-objects even if a
- *            checksum already exists in iCAT. This keyWd has no value.
+ *        \li "verifyChksum" - verify the checksum value.
+ *        \li "forceChksum" - checksum data objects even if a
+ *            checksum already exists in the catalog. This keyWd has no value.
```

Comment:
Could change keyWd to keyword?

Not required though.

Reply:
ignoring for now.   i saw a couple other consistency things in there that i didn't address.  will get them later.